rts/lra-examples/run.sh

```diff
@@ -122,15 +122,15 @@ function test_recovery {
 
 echo "===== Running qickstart $qsname in directory $PWD"
 
-if [[ "$last" != "coordinator" ]]; then
+if [[ "$last" != "embedded" ]]; then
```

Comment:
I would be tempted to make this more precise and say if this is the "participant", I would also consider to make the else clause below something like `elif [[ $last" = "embedded" ]]`

Reply:
I had a quick look and the other two folders in https://github.com/jbosstm/quickstart/tree/master/rts/lra-examples (lra-jwt and lra-coordinator) don't seem to trigger this script (run.sh) - at least in their top level pom.xml - so this is advisory

Reply:
Thanks @tomjenkinson, I think it is a good idea. In the else statement I echo and error message and then I exit with error code.